### PR TITLE
Update util scripts.

### PR DESCRIPTION
**Pull request by xingweitian**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,5 @@
 # Constant Value Inference (Interval Inference)
 
-[![build-status](https://travis-ci.org/txiang61/cast_checker.svg?branch=master)](https://travis-ci.org/txiang61/cast_checker)
```
Comment (wmdietl) [inline README.md:3]: Wouldn't this be nice, when change to `value-inference`?

File: README.md [hunk 2]
```diff
@@ -12,14 +10,8 @@ This project is developed based on [Checker Framework](http://types.cs.washingto
 
 - txiang61/checker-framework
 - txiang61/checke-framework-inference
-- jsr308-langtools
-- annotation-tools
```
Comment (wmdietl) [inline README.md:16]: annotation-tools should stay? Maybe change to `opprop/annotation-tools`?

File: .travis.yml [hunk 3]
```diff
@@ -1,13 +1,10 @@
...
-  apt:
-    packages:
-      - ant-optional
-
+dist: trusty
 jdk:
   - oraclejdk8
+
+install:
+  - pip install --user  -r ./.travis-requirements.txt
```
Comment (wmdietl) [inline .travis.yml]: Should this PR add `.travis-requirements.txt`?

File: README.md (hunk 1 above)
Comment (xingweitian) [reply README.md:3]: Yes, if we decide to go with Travis, then it is nice to keep this.

File: README.md (hunk 2 above)
Comment (xingweitian) [reply README.md:16]: Building CF/CFI will automatically download `annotation-tools` and other dependencies. What do you think?

File: .travis.yml (hunk 3 above)
Comment (xingweitian) [reply .travis.yml]: If we decide to do CI in a docker container (like what CF does), then this is unnecessary. But if we use a third-party provided OS (like ontology and security-demo using ubuntu from Travis), then `.travis-requirements.txt` should be kept. I am fine with both ways. What do you think?

**Pull request merged**